transport_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestInitTransportWithDefault(t *testing.T) {
+	roundTripper, err := initTransport("")
+	if err != nil {
+		t.Errorf("initTransport(\"\") is legitimate. Should return http.DefaultTransport. But returned %v", err)
```

Comment:
nit, a more idiomatic test failure would be:
```
t.Errorf("want err to be nil, but got %v", err)
```

Reply:
If I understand the logic correctly, here we should `return` too?

Reply:
yes :+1: